internal/server/data/data.go

```diff
@@ -247,18 +277,27 @@ func handleError(err error) error {
 }
 
 func delete[T models.Modelable](db *gorm.DB, id uid.ID) error {
+	if hasOrgID(new(T)) {
+		org := OrgFromContext(db.Statement.Context)
+		db = db.Where("organization_id = ?", org.ID)
```

Comment:
This looks like it should use the selector too (like in `get` and `list`)
```suggestion
	if hasOrgID(new(T)) {
		db = ByOrgID(OrgFromContext(db.Statement.Context).ID)(db)
	}
```

Reply:
Ya, may as well be consistent. All the selectors are going to be removed soon, but doesn't hurt to be consistent in the interim.